content/en/docs/instrumentation/java/automatic/agent-config.md

```diff
@@ -4,7 +4,12 @@ linkTitle: Configuration
 weight: 10
 spelling:
   cSpell:ignore akka autoconfiguration Autoconfiguration Dotel HSET javaagent
-  cSpell:ignore serverlessapis Servlet servlet
+  cSpell:ignore serverlessapis Servlet servlet Customizer classloaders logback
```

Comment:
Do we actually want both Servlet and servlet in this list? Only "servlet" should be enough, right?

Reply:
```suggestion
  cSpell:ignore serverlessapis servlet Customizer classloaders logback
```